pombola/core/models.py

```diff
@@ -767,6 +768,24 @@ def is_ongoing(self):
             now_approx = ApproximateDate(year=now.year, month=now.month, day=now.day)
             return now_approx <= self.ended
 
+    def __shorten_name(self):
+
+        name = self.name
+
+        strip_phrases = [
+            'Portfolio Committee on',
+            'Standing Committee on',
+        ]
+
+        for phrase in strip_phrases:
+            name = re.sub('(?i)' + re.escape(phrase), '', name)
+
+        return name.strip()
+
+    def save(self, *args, **kwargs):
+        self.short_name = self.__shorten_name()
```

Comment:
Should this happen only if `not self.short_name`, so that people can tweak it manually?

Reply:
I contemplated this approach, but then made `short_name` a non-editable field in favour of maintaining an entirely programmatical approach. The reasoning being that otherwise we would have had to create an entirely new property to access `short_name` based on if it was set or not (and then also created `get_` and `set_` methods for a hidden `_short_name` or similar), which would have made sorting awkward.